src/packages/router/route/index.js

```diff
@@ -55,65 +57,44 @@ class Route extends FreezeableSet<Action<any>> {
           dynamicSegments
         });
 
+        const staticPath = getStaticPath(path, dynamicSegments);
+
+        const defaultParams = defaultParamsFor({
```

Comment:
Using a frozen object for `defaultParams` should result in a small perf boost. Previously `defaultParams` was computed on every request, now it is stored as a member of the `Route` instance.